poi/src/test/java/org/apache/poi/hssf/model/TestFormulaParser.java

```diff
@@ -1357,8 +1357,8 @@ void testUnionOfFullCollFullRowRef() throws IOException {
                 UnionPtg.class
         );
 
-        wb.createSheet("Sh't1");
-        ptgs = parse("'Sh''t1'!$A:$A,'Sh''t1'!$1:$4", wb);
```

Comment:
could you add a new test and leave this one as is? It must be useful to keep the test for `"Sh't1"`.

Reply:
Yes, I'll do it

Reply:
Done